wildlifelicensing/apps/applications/pdf.py

```diff
@@ -0,0 +1,182 @@
+from io import BytesIO
+
+from reportlab.lib import enums
+from reportlab.lib.pagesizes import A4
+from reportlab.platypus import BaseDocTemplate, PageTemplate, Frame, Paragraph, Spacer, Table, TableStyle
+from reportlab.lib.styles import getSampleStyleSheet, ParagraphStyle
+
+
+PAGE_WIDTH, PAGE_HEIGHT = A4
+
+DEFAULT_FONTNAME = 'Helvetica'
+BOLD_FONTNAME = 'Helvetica-Bold'
+ITALIC_FONTNAME = 'Helvetica-Oblique'
+BOLD_ITALIC_FONTNAME = 'Helvetica-BoldOblique'
+
+VERY_LARGE_FONTSIZE = 16
+LARGE_FONTSIZE = 14
+MEDIUM_FONTSIZE = 12
+SMALL_FONTSIZE = 10
+
+PARAGRAPH_INDENT = 15
+PARAGRAPH_BOTTOM_MARGIN = 2
+
+SECTION_BUFFER_HEIGHT = 10
+
+HEADER_MARGIN = 10
+
+PAGE_MARGIN = 20
+PAGE_TOP_MARGIN = 200
+
+DATE_FORMAT = '%d/%m/%Y'
+
+MISSING_VALUE_PLACEHOLDER = 'Not specified'
+MISSING_FILE_PLACEHOLDER = 'No file attachment provided'
+
+
+styles = getSampleStyleSheet()
+styles.add(ParagraphStyle(name='ApplicationTitle', fontName=BOLD_FONTNAME, fontSize=VERY_LARGE_FONTSIZE,
+                          spaceAfter=HEADER_MARGIN, alignment=enums.TA_CENTER))
+styles.add(ParagraphStyle(name='ApplicationVariantsTitle', fontName=BOLD_FONTNAME, fontSize=LARGE_FONTSIZE,
+                          spaceAfter=PARAGRAPH_BOTTOM_MARGIN * 2, alignment=enums.TA_CENTER))
+
+styles.add(ParagraphStyle(name='BoldLeft', fontName=BOLD_FONTNAME, fontSize=SMALL_FONTSIZE, alignment=enums.TA_LEFT))
+styles.add(ParagraphStyle(name='Left', alignment=enums.TA_LEFT))
+
+
+def _create_application(application_buffer, application):
+    every_page_frame = Frame(PAGE_MARGIN, PAGE_MARGIN, PAGE_WIDTH - 2 * PAGE_MARGIN, PAGE_HEIGHT - 2 * PAGE_MARGIN,
+                             id='EveryPagesFrame')
+    every_page_template = PageTemplate(id='EveryPages', frames=every_page_frame)
```

Comment:
Isn't `frames` supposed to be a list as its plural name indicates, like `pageTemplates` below.